modules/tinymce/src/plugins/table/test/ts/browser/command/MergeCellCommandTest.ts

```diff
@@ -3,29 +3,50 @@ import { UnitTest } from '@ephox/bedrock-client';
 import { LegacyUnit, TinyLoader } from '@ephox/mcagar';
 
 import Editor from 'tinymce/core/api/Editor';
+import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
+import Tools from 'tinymce/core/api/util/Tools';
 import Plugin from 'tinymce/plugins/table/Plugin';
 import SilverTheme from 'tinymce/themes/silver/Theme';
 
 UnitTest.asynctest('browser.tinymce.plugins.table.command.MergeCellCommandTest', (success, failure) => {
   const suite = LegacyUnit.createSuite<Editor>();
+  type MergeCellTest = {
```

Comment:
```suggestion
  interface MergeCellTest {
```
And also readonly for the properties please.